bin/hdfcoinc/pycbc_multiifo_dtphase

```diff
@@ -0,0 +1,169 @@
+#!/bin/env python
+""" Create a file containing the phase and amplitude, 
+correlations between two detectors by
+doing a simple monte-carlo
+
+Output is the relative SNR, time, and phase as compared to a reference
```

Comment:
is it relative SNR or relative amplitude?

Reply:
Should be amplitude, it's only scaled to an "SNR" to apply some minimal uncertainties.